### ISSUE TITLE
Not compatible with Angular 10.1

### ISSUE THREAD

**Issue by juaal12** [rxweb/rxweb#406]
# Describe the bug

At least "@rxweb/reactive-form-validators" is not compatible with Angular 10.1 and produces a build error. This issue is supossed to be resolved on 2.1.1 version but still not working. Related issue #398 

```
ERROR in node_modules/@rxweb/reactive-form-validators/services/form-control.d.ts:46:5 - error TS2610: 'dirty' is defined as an accessor in class 'FormControl', but is overridden here in 'RxFormControl' as an instance property.

46     dirty: boolean;
```

# Expected Behavior

It should not produce a build error.

# To Reproduce

Add "@rxweb/reactive-form-validators" to a Angular 10.1 project and run ng build

# Package Version
@rxweb/reactive-form-validators@2.1.1

**Comment by vrait35** [comment]
returns the same error as in version 2.1.0

**Comment by ajayojha** [comment]
@juaal12 & @vrait35 Sorry, the issue is fixed in the 2.1.2 release. Please update the package.
